HW03/aleksei_zhuchkov/spec/hw_03_t_02_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative '../aleksei_zhuchkov_hw_03_t_02'
+
+RSpec.describe 'Homework3' do
+  let(:obj_hw3) { Homework3.new }
+
+  describe 'Testing basics things' do
+    it 'return value is Array type' do
+      expect(obj_hw3.task2('')).to be_an_instance_of(Array)
+    end
+
+    it 'empty input - return empty array' do
+      expect(obj_hw3.task2('')).to eq([])
+    end
+
+    context 'when input-data was wrong' do
+      let(:log) do
+        <<~LOGS
+          hello world
+          first test is more difficult
+          try to create some intresting there
+        LOGS
+      end
+
+      it 'return empty array if input-data format was wrong' do
```

Comment:
your if is used in context
```suggestion
      it 'returns an empty array' do
```

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/1832f4c0cf37bf53b5b4d689667af084de8d4ab5